tests/e2e/v3_curl_test.go

```diff
@@ -366,6 +366,21 @@ func testV3CurlResignMissiongLeaderKey(cx ctlCtx) {
 	}
 }
 
+func TestV3CurlMaintenanceAlarmMissiongAlarm(t *testing.T) {
```

Comment:
"Missiong" is how it's spelled in all the other tests in this file, so I decided to be consistent rather than correct 🤷